tests/core/pyspec/eth2spec/test/altair/merkle/test_single_proof.py

```diff
@@ -8,8 +8,14 @@
 
 @with_phases([ALTAIR])
```

Comment:
I would default to doing `@with_all_phases_except([PHASE0])` and if the beaconstate changes enough in subsequent phase to break this test, it will alert us to either make this more modular or to add additional merkle tests for new phase

Reply:
Good point, but more of a separate issue that already existed before this PR.

Reply:
https://github.com/ethereum/consensus-specs/pull/2635